packages/matter.js/src/schema/BitmapSchema.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 import { ByteArray } from "../util/ByteArray.js";
+import { Merge } from "../util/index.js";
```

Comment:
not use index import please

Reply:
OK I went ahead and removed index imports from all files whether they were part of the PR or not, except for tests, I thought those ones might've been intentional.

Reply:
thank you! ... Also for tests most likely not intentional :-) but all fine. we can fix separately